completion/latex-document.cwl

```diff
@@ -3,18 +3,21 @@
 # tbraun/2006-08-03 removed dup inserted by me ...
 # tbraun/2007-02-24 added left/right variants
 # dbitouze/2015-02-21 obsolete font style commands made unusual
-# Matthew Bertucci 2024/11/01 release
+# Matthew Bertucci 2025/06/01 release
 
 #include:latex-dev
 
 \abstractname#*
 \addcontentsline{ext}{type}{entry%text}#*
 \Alph{counter}#*
+\Alph*#*
```

Comment:
Maybe accepting `*` as a built-in special counter name instead?

Reply:
That’s up to @sunderme